static/templates/reports/candidate.hbs

```diff
@@ -111,9 +111,6 @@
     {{#panelRow "Total loans received"}}
       {{{currency total_loans_received_period}}}
     {{/panelRow}}
-    {{#panelRow "Total loans received from candidates"}}
```

Comment:
I removed it because there's an inconsistency between house-senate reports and presidential reports in what the field is named, and I figured easier to just remove it for the time being since we haven't totally fleshed out what the rules are for what is included in these panels. But I can add it back if you think it's important.